src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -1768,9 +1731,9 @@ def _delete_credential(graph_object, remove_password_func, patch_function, key_i
                 'keyCredentials': key_creds
             }
             patch_function(graph_object[ID], body)
-        print(graph_object)
-        raise CLIError("'{}' doesn't exist in graph object '{}'.".format(
-            key_id, graph_object[ID]))
+        else:
+            raise CLIError("No key credential found with keyId as '{}' in graph object '{}'.".format(
```

Comment:
Use the same error message as https://docs.microsoft.com/en-us/graph/api/application-removekey?view=graph-rest-1.0&tabs=http